package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@holochain/conductor-api",
-  "version": "0.0.1-dev.15",
+  "version": "0.0.1-dev.16",
```

```suggestion
  "version": "0.0.1-dev.15",
```

Unless you're doing things differently, I don't think you want to bump the version in the PR, `npm version` will do this automatically for you.

Also, it seems like next release could be a full patch release. I think now that we've hit the big-time we're ready to officially release `0.0.1` :smile: